Documentation/WritingManPages.md

```diff
@@ -0,0 +1,89 @@
+# Guidelines for writing SerenityOS manual pages (manpages)
+
+## What does and does not belong in manual pages
+
+The SerenityOS manpages are the primary documentation for SerenityOS itself. Different from the documentation pages, it is not primarily intended for SerenityOS developers. If the question of "Would this information be useful to view within the OS?" can be answered with "Yes", that probably means the information belongs in a manual page.
```

Comment:
Worth a reference to some authoritative source on manpage style from Linux/BSD/Unix etc?

Reply:
The only *stylistic* information I can find is with Linux and FreeBSD, so I'll link those.